### DISCUSSION TITLE
[FR]: want a column that displays the dataview's file.folder information

### DISCUSSION THREAD

**Discussion by raggbal** [RafaelGB/obsidian-db-folder#867] [Q&A] [answered]
### Present your request

In dataview, it is possible to display the relative path of the parent folder of a file simply by specifying file.folder.
I'd like to do the same thing, but I didn't know how to do it.

ex.

```dataview 
TABLE file.folder
```

### For which platform do you request this request??

Desktop

**Comment by bithead2k** [comment, marked as answer]
In the database query, add this:

```sql
,file.folder as folder FROM "journals"
```

and then use "add empty column" to add a column named "folder".   Like magic, it will be populated with the folder name, which in my case was:

| File | folder | + |
|:------|:---------|----|
|2022| journals/2022 | |
|2023| journals/2023 | |
...snip

**Comment by raggbal** [comment]
It worked! Like magic!
Thank you so much  :)
